compiler/src/tests/cxxfrontend.cc

```diff
@@ -964,8 +964,8 @@ class MiniGlueVisitor : public Visitor
         else
             sym = func->searchLabel(s->ident, s->loc);
         sym->statement->accept(this);
-        if (sym == func->returnLabel && func->fensure != NULL)
-            func->fensure->accept(this);
+        if (sym == func->returnLabel && func->fensure() != NULL)
```

Comment:
This is probably a breaking change for ldc and gdc glue code?

@ibuclaw @kinke ?